components/create_post/create_post.jsx

```diff
@@ -376,23 +380,47 @@ export default class CreatePost extends React.Component {
             return;
         }
 
-        let message = this.state.message;
-        let ignoreSlash = false;
-        const serverError = this.state.serverError;
+        const time = Utils.getTimestamp();
+        const userId = currentUserId;
+        let post = {};
+        post.file_ids = [];
+        post.message = this.state.message;
+        post.channel_id = channelId;
+        post.pending_post_id = `${userId}:${time}`;
+        post.user_id = userId;
+        post.create_at = time;
+        post.parent_id = this.state.parentId;
```

Comment:
Can you remove this line? It doesn't look like it ever does anything since `parentId` doesn't exist in `this.state`, but any references to parent ID worry me since that hasn't been used for so long.